### ISSUE TITLE
Error in AdtClient.js library

### ISSUE THREAD

**Issue by maxx1e** [pfefferf/ui5-nwabap-deployer#24]
Last update brought the following issues while executing: 

```
info builder:builder Building project Z_CREATE_INC not including dependencies...
info builder:builder 🛠   (1/1) Building project Z_CREATE_INC
info builder:builder application Z_CREATE_INC 🔨 (1/9) Running task escapeNonAsciiCharacters...
info builder:builder application Z_CREATE_INC 🔨 (2/9) Running task replaceCopyright...
info builder:builder application Z_CREATE_INC 🔨 (3/9) Running task replaceVersion...
info builder:builder application Z_CREATE_INC 🔨 (4/9) Running task generateFlexChangesBundle...
info builder:builder application Z_CREATE_INC 🔨 (5/9) Running task generateComponentPreload...
info builder:builder application Z_CREATE_INC 🔨 (6/9) Running task createDebugFiles...
info builder:builder application Z_CREATE_INC 🔨 (7/9) Running task uglify...
info builder:builder application Z_CREATE_INC 🔨 (8/9) Running task generateVersionInfo...
info builder:builder application Z_CREATE_INC 🔨 (9/9) Running task ui5-task-nwabap-deployer...
(node:386) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'data' of undefined-task-nwabap-deployer...
    at AdtClient.sendRequest (/usr/node/home/node_modules/ui5-nwabap-deployer-core/lib/AdtClient.js:186:103)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:386) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 44)
(node:386) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by pfefferf** [comment]
The error occurred because in case no connection can be established to the backend (ECONNREFUSED) the error object has no response object which is accessed in the error handler.
I fixed that in ui5-nwabap-uploader-core@1.0.8. 
Please can you check it.

**Comment by maxx1e** [comment]
thnx, all is good!
